Lavinia-api/Data/CustomValidation.cs

```diff
@@ -15,22 +15,43 @@ public static class CustomValidation
         /// <param name="checkedModels">A set of models that has been checked already.</param>
         public static void ValidateCountry(Country country, HashSet<int> checkedModels)
         {
-            if (country == null) throw new ArgumentException("Country cannot be null.");
+            if (country == null)
+            {
+                throw new ArgumentException("Country cannot be null.");
+            }
 
             if (country.InternationalName == null)
+            {
                 throw new ArgumentException("Country.InternationalName cannot be null.");
+            }
+
             if (country.InternationalName.Length < 3)
+            {
                 throw new ArgumentException("Country.InternationalName cannot be shorter than 3 characters.");
-            if (country.CountryCode == null) throw new ArgumentException("Country.ShortName cannot be null.");
```

Comment:
This change does not seem to match the description of the commit.

Reply:
The git lines are misaligned, if you look further down you can see that the change is simply adding curly braces to the if.